### ISSUE TITLE
Feature request : Turn off LED while screen on

### ISSUE THREAD

**Issue by VinhNgT** [GravityBox/GravityBox#1888]
The front notification LED, not the one on the back

**Comment by C3C0** [comment]
Not sure what device/system you have but turning off LED when user starts interacting with a device is default behavior in Android.

**Comment by VinhNgT** [comment]
Yes it is, but some how it just not works on my phone (Galaxy S5, LineageOS 15.1). Every thing was fine on stock rom so no hardware issue there. And also it seems like there some other people on google is having the same problem too.

**Comment by C3C0** [comment]
I am sorry but GravityBox is not designed to be used on custom ROMs. Supporting such ROMs is unmanageable as they often change and I simply cannot keep up with the changes. The best practice is to avoid using custom-ROM like modules on such ROMs and rather ask devs to implement features of interest the proper way - at source code level.

**Comment by devanteweary** [comment]
I'd actually prefer to have an option to leave the LED on until I clear the notification, even with the screen on.
